api/v1beta1/cinderapi_types.go

```diff
@@ -113,7 +113,7 @@ func init() {
 	SchemeBuilder.Register(&CinderAPI{}, &CinderAPIList{})
 }
 
-// IsReady - returns true if service is ready to serve requests
+// IsReady - returns true if CinderAPI is reconciled successfully
 func (instance CinderAPI) IsReady() bool {
-	return instance.Status.ReadyCount == instance.Spec.Replicas
```

Comment:
So current IsReady requires that requested number of pods are ready, but (Deployment)ReadyCondition is set to true if at least one pod is available.

My understanding is these services should work as long as at least one pod is available but please correct me if there is any additional requirements.